src/Mod/Assembly/JointObject.py

```diff
@@ -1494,19 +1494,35 @@ def adaptUi(self):
             self.form.distanceSpinbox.setProperty("minimum", float("-inf"))
             self.form.distanceSpinbox2.setProperty("minimum", float("-inf"))
 
-        if jType in JointUsingOffset:
-            self.form.offsetLabel.show()
-            self.form.offsetSpinbox.show()
-        else:
+        if self.form.advancedOffsetCheckbox.isChecked():
+            self.form.offset1Label.show()
+            self.form.offset2Label.show()
+            self.form.offset1Button.show()
+            self.form.offset2Button.show()
+
             self.form.offsetLabel.hide()
             self.form.offsetSpinbox.hide()
-
-        if jType in JointUsingRotation:
-            self.form.rotationLabel.show()
-            self.form.rotationSpinbox.show()
-        else:
             self.form.rotationLabel.hide()
             self.form.rotationSpinbox.hide()
```

Comment:
I'm not sure if hiding the simple offsets is necessary in that case? The checkbox will from under the mouse when cliecked, which is non ideal. These controls are synced so it should not be a problem to show both at the same tame if some one opts to see advanced offsets.

Reply:
Good question. My initial intent was that the advanced hide the simple so that user can see they are two face of the same coin. But I'm open to something else. Having both at the same time is possible indeed.